src/greeklt/greek_iso_transliteration.py

```diff
@@ -0,0 +1,272 @@
+def greek_iso_transliteration(string: str):
```

Comment:
I would rather name this `greek_elot_transliteration`

Reply:
Fixed!